crates/cxx-qt-gen/src/generator/rust/method.rs

```diff
@@ -179,7 +178,6 @@ mod tests {
             &generated.cxx_mod_contents[0],
             quote! {
                 extern "Rust" {
-                    #[doc(hidden)]
```

Comment:
These tests, as well as the expected output test probably still need updating.
I assume that's also why CI fails.

Reply:
Done